wireapp/wire-android
# feat: Display reset session system message (AR-2838)

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-2838" title="AR-2838" target="_blank"><img alt="Subtask" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10816?size=medium" />AR-2838</a>  display session reset system message
  </td></table>
  <br />
 






----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Description

This PR aims to display  reset session system message, on sender side, when  a user reset his crypto session.

### Dependencies (Optional)

Needs to be merged with :
[Feat: Receive reset session system message
](https://github.com/wireapp/kalium/pull/1237
)
Needs releases with:


### Testing

#### Test Coverage (Optional)


#### How to Test

B have some messages, sent from A, that failed to be decrypted.
When B try to reset his crypto session, A should see a system message informing that B have some issue with decryption and he decided to reset his session.

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments 

![Wire 2022-12-15 at 6_00 PM](https://user-images.githubusercontent.com/18124919/207924184-4559bdda-cefa-49d2-bdae-ee1286a9ac9e.png)
(Optional)


_Attachments like images, videos, etc. (drag and drop in the text box)_ 



----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
